src/components/MemberInfo/MemberInfo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './MemberInfo.css'
+import { Icon } from '@iconify/react';
+
```

Comment:
i feel like the member info component will only be used for the Group Page, so instead of putting the MemberInfo.js and MemberInfo.css in the src/components folder, make a new folder inside src/pages/Group called components and add the files there. So the pathing will look like "src/pages/Group/components/MemberInfo/MemberInfo.js" and "src/pages/Group/components/MemberInfo/MemberInfo.css"